code/game/objects/structures/tables_racks.dm

```diff
@@ -638,42 +638,14 @@
 
 /obj/structure/surface/table/reinforced/tacmap
```

Comment:
probably just leave the table out entirely/replace it with one of our existing maptables if you're not interested in fully implementing it.